datafusion/expr/src/built_in_function.rs

```diff
@@ -687,6 +691,9 @@ impl BuiltinScalarFunction {
             }
             BuiltinScalarFunction::CurrentDate => Ok(Date32),
             BuiltinScalarFunction::CurrentTime => Ok(Time64(Nanosecond)),
+            BuiltinScalarFunction::CurrentTimestamp => {
+                Ok(Timestamp(Nanosecond, Some("+00:00".into())))
```

Comment:
Maybe we could combine this with the case for `Now` to show they are the same function (just aliased)